bulletin-buddy-client-service/bulletin-buddy-client-service-web/src/app/overview-page/overview-page.component.ts

```diff
@@ -10,13 +10,32 @@ import { PostsService } from '../posts.service';
 })
 export class OverviewPageComponent implements OnInit {
   public posts: Post[] = [];
+  public filtered_posts: Post[] = [];
```

Comment:
For consistency, I would stick to the conventions when it comes to casing (in this case, camelCase instead of snake_case) so we everything is uniform

Reply:
Other than that, looks good to me. But I don't know Angular that well, so maybe @skrhakv can do a better review